data-mermaid/mermaid-webapp
# Fix:[M916 & M917] duplicate MR modal improvements.

**Changes**
- Fix for M916 Resolve duplicate MR names https://trello.com/c/jfL4O12z/916-resolve-duplicate-mr-names
- Fix for M917 Similar SE with different MR warning https://trello.com/c/lhCB1N4w/917-similar-se-with-different-mr-warning.
- Update TableRowItem styles, and remove unused styles.
- Change some language texts.

**To test for M916:**
1) Have 2 MRs with same name in managements table.
2) Have a submitted record with one of the MRs .
3) Create a collect record, and select another MR with the same name as the one in the submitted record in step 2.
4) Save -> Validate.
5) The warning `Another Management Regime is similar to this one.` should appear with `Resolve` button at the side.
_Note:_ Resolve duplicate MR modal functions should work even though there might be bug mentioned in trello ticket M916

**To test for M917:**
1) Have a submitted record with site called `Site A` (example name, you could import from copy sites modal, or create a new one)
2) Create a collect record with any sample unit, select exact site name as submitted record from step 1, select MR (choose the one different from step 1 submitted record).
3) The warning `Other sample events at this site have a different management regime. Go to Management Regime Overview` should appear with `Ignore` checkbox.
